akka-docs/.history

```diff
@@ -1 +0,0 @@
-exit
```

Comment:
@ktoso @patriknw I'm not sure, it was there in the commit I originally based my PR on so I removed it, and the file is already ignored. I'm not sure what happened during the rebase, but I don't see the file in my repo...
I could try to amend it if it's still there.